server/protoboards.go

```diff
@@ -0,0 +1,76 @@
+package server
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/bouk/httprouter"
+	"github.com/influxdata/chronograf"
+)
+
+type protoboardLinks struct {
+	Self string `json:"self"`
+}
+
+type protoboardResponse struct {
+	chronograf.Protoboard
+	Links protoboardLinks `json:"links"`
+}
+
+func newProtoboardResponse(protoboard chronograf.Protoboard) protoboardResponse {
+	httpAPIProtoboards := "/chronograf/v1/protoboards"
+	selfLink := fmt.Sprintf("%s/%s", httpAPIProtoboards, protoboard.ID)
+
+	return protoboardResponse{
+		Protoboard: protoboard,
+		Links: protoboardLinks{
+			Self: selfLink,
+		},
+	}
+}
+
+type getProtoboardsResponse struct {
+	Protoboards []protoboardResponse `json:"protoboards"`
+}
+
+// Protoboards retrieves all protoboards from store
+func (s *Service) Protoboards(w http.ResponseWriter, r *http.Request) {
+
+	ctx := r.Context()
+	protoboards, err := s.Store.Protoboards(ctx).All(ctx)
+	if err != nil {
+		Error(w, http.StatusInternalServerError, "Error loading protoboards", s.Logger)
```

Comment:
We don't include the error that we encountered?

Reply:
copied from layouts. want us to just pass through whatever error?